Samples/iOS-Swift/iOS-Swift-UITests/UserFeedbackUITests.swift

```diff
@@ -0,0 +1,178 @@
+//swiftlint:disable todo
+
+import XCTest
+
+class UserFeedbackUITests: BaseUITest {
+    override var automaticallyLaunchAndTerminateApp: Bool { false }
+    
+    override func setUp() {
+        super.setUp()
+        app.launchArguments.append(contentsOf: [
+            "--io.sentry.iOS-Swift.auto-inject-user-feedback-widget",
+            "--io.sentry.iOS-Swift.user-feedback.all-defaults",
+            "--io.sentry.feedback.no-animations"
+        ])
+        launchApp()
+    }
+    
+    func testSubmitFullyFilledForm() throws {
+        widgetButton.tap()
+        
+        nameField.tap()
+        nameField.typeText("Andrew")
+        
+        emailField.tap()
+        emailField.typeText("andrew.mcknight@sentry.io")
+        
+        messageTextView.tap()
+        messageTextView.typeText("UITest user feedback")
+        
+        app.staticTexts["Send Bug Report"].tap()
+        
+        // displaying the form again ensures the widget button still works afterwards; also assert that the fields are in their default state to ensure the entered data is not persisted between displays
+        
+        widgetButton.tap()
+        
+        // the placeholder text is returned for XCUIElement.value
+        XCTAssertEqual(try XCTUnwrap(nameField.value as? String), "Your Name")
+        XCTAssertEqual(try XCTUnwrap(emailField.value as? String), "your.email@example.org")
+        
+        // the UITextView doesn't hav a placeholder, it's a label on top of it. so it is actually empty
+        XCTAssertEqual(try XCTUnwrap(messageTextView.value as? String), "")
+    }
+    
+    func testSubmitWithNoFieldsFilled() throws {
+        widgetButton.tap()
+        
+        app.staticTexts["Send Bug Report"].tap()
+        
+        XCTAssert(app.staticTexts["Error"].exists)
+        
+        app.buttons["OK"].tap()
+    }
+    
+    func testSubmitWithOnlyRequiredFieldsFilled() {
+        widgetButton.tap()
+        
+        messageTextView.tap()
+        messageTextView.typeText("UITest user feedback")
+        
+        app.staticTexts["Send Bug Report"].tap()
+        
+        XCTAssert(widgetButton.waitForExistence(timeout: 1))
+    }
+    
+    func testSubmitOnlyWithOptionalFieldsFilled() throws {
+        widgetButton.tap()
+        
+        nameField.tap()
+        nameField.typeText("Andrew")
+        
+        emailField.tap()
+        emailField.typeText("andrew.mcknight@sentry.io")
+        
+        app.staticTexts["Send Bug Report"].tap()
+        
+        XCTAssert(app.staticTexts["Error"].exists)
+        
+        app.buttons["OK"].tap()
```

Comment:
`m`: I think it would be great to continue the test to validate that you can submit the form after you close the error message. Either adding this in this test or adding an extra one is both fine for me.

Reply:
I'll look at doing this in the next PR since there is more testing stuff there 👍🏻